### PR TITLE
Add migration instructions for configuration files

### PR DESCRIPTION
**Pull request by kmakiela**
This PR introduces a first draft of the migration guide. For now it includes changes in the configuration files only.

### CODE REVIEW

File: guides/migrations/2.0.x_2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+### OpenAPI
+MongoosePush can expose OpenAPI specification if enabled. Config entry may look like this:
+```elixir
+config :mongoose_push,
+  openapi: [
+    expose_spec: {:system, :boolean, "PUSH_OPENAPI_EXPOSE_SPEC", false},
+    expose_ui: {:system, :boolean, "PUSH_OPENAPI_EXPOSE_UI", false}
+  ]
+  ```
+`PUSH_OPENAPI_EXPOSE_SPEC` enables OpenAPI endpoint support and `PUSH_OPENAPI_EXPOSE_UI` enables SwaggerUI. Both these options are disbaled by default. For more details refer to the `configuration` guide.
```
Comment (jan-ciesla) [inline guides/migrations/2.0.x_2.1.0.md:112]:
```suggestion
`PUSH_OPENAPI_EXPOSE_SPEC` enables OpenAPI endpoint support and `PUSH_OPENAPI_EXPOSE_UI` enables SwaggerUI. Both these options are disabled by default. For more details refer to the `configuration` guide.
```
When refering to `configuration` guide, do you mean any specific part of it? Maybe we could add a link that directs to the relevant part.

File: guides/migrations/2.0.x_2.1.0.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+MongoosePush does not use `Elixometer` for its metrics anymore. Metrics are provided using `Telemetry` and `elixometer` config entry should be removed.
+Old `elixometer` entry:
+```elixir
+config :elixometer,
+  reporter: :exometer_report_tty,
+  env: Mix.env(),
+  metric_prefix: "mongoose_push"
+  ```
+
+There is no config entry necessary for `Telemetry` metrics. To learn more about the new metrics check out the `metrics` guide.
```
Comment (jan-ciesla) [inline guides/migrations/2.0.x_2.1.0.md:124]: and here we could add a link as well.

**Comment by kmakiela** [comment]
Additionally changed the relative links to the other parts of documentation in MD files as it was not working properly

**Comment by kmakiela** [comment]
This PR encountered some bugs with GitHub and is being closed.

**Pull request closed without merging**